src/components/ModalDrawer.js

```diff
@@ -24,25 +25,28 @@ const drawer = {
 }
 
 const ModalDrawer = ({ isOpen, onDismiss, width = 450, children, ...props }) => {
+  Utils.useLabelAssert('ModalDrawer', props)
+
   return h(Transition, {
     in: isOpen,
     timeout: { exit: 200 },
     appear: true,
     mountOnEnter: true,
     unmountOnExit: true
   }, [transitionState => h(RModal, {
+    aria: { label: props['aria-label'], labelledby: props['aria-labelledby'], modal: true },
+    ariaHideApp: false,
     parentSelector: () => document.getElementById('modal-root'),
     isOpen: true,
     onRequestClose: onDismiss,
     style: { overlay: drawer.overlay(transitionState), content: { ...drawer.container(transitionState), width } },
-    ariaHideApp: false,
     ...props
   }, [children])])
 }
 
-export const withModalDrawer = ({ width } = {}) => WrappedComponent => {
+export const withModalDrawer = ({ width, ...modalProps } = {}) => WrappedComponent => {
```

Comment:
do we need the spread operator here?

Reply:
It is more accurate to the goal (which is as individual props), but not strictly speaking necessary. My inclination is to keep it with the spread operator, since that's more indicative of what it is trying to do.